ihmc-graphics/src/libgdx/java/us/ihmc/rdx/RDXModelViewer.java

```diff
@@ -19,7 +19,7 @@ public void create()
             baseUI.getPrimaryScene().addCoordinateFrame(1.0);
 
             Model model = RDXModelLoader.load(modelFileName);
-            baseUI.getPrimaryScene().addModelInstance(new ModelInstance(model));
+            baseUI.getPrimaryScene().addModelInstance(new RDXModelInstance(model));
```

Comment:
Using the constructor in RDXModelInstance includes conversion to PBR compatible materials.